apm-agent-core/src/test/java/co/elastic/apm/report/serialize/DslJsonSerializerTest.java

```diff
@@ -68,15 +68,21 @@ void testLimitStringValueLength() throws IOException {
             longValue.append('0');
         }
 
+        StringBuilder sqlValue = new StringBuilder(DslJsonSerializer.MAX_SQL_VALUE_LENGTH + 1);
+        for (int i = 0; i < DslJsonSerializer.MAX_SQL_VALUE_LENGTH + 1; i++) {
+            sqlValue.append('0');
+        }
         serializer.jw.writeByte(JsonWriter.OBJECT_START);
         serializer.writeField("stringBuilder", longValue);
         serializer.writeField("string", longValue.toString());
         serializer.writeLastField("lastString", longValue.toString());
+        serializer.writeFieldSQLStatement("statement", sqlValue.toString());
         serializer.jw.writeByte(JsonWriter.OBJECT_END);
         final JsonNode jsonNode = objectMapper.readTree(serializer.jw.toString());
         assertThat(jsonNode.get("stringBuilder").textValue()).hasSize(DslJsonSerializer.MAX_VALUE_LENGTH).endsWith("…");
         assertThat(jsonNode.get("string").textValue()).hasSize(DslJsonSerializer.MAX_VALUE_LENGTH).endsWith("…");
         assertThat(jsonNode.get("lastString").textValue()).hasSize(DslJsonSerializer.MAX_VALUE_LENGTH).endsWith("…");
+		assertThat(jsonNode.get("statement").textValue()).hasSize(DslJsonSerializer.MAX_SQL_VALUE_LENGTH).endsWith("…");
```

Comment:
use spaces

Reply:
If i understand correctly, you would want to check if the value endsWith ("  "). Is that the right understanding?

Reply:
ok....what does "use spaces" mean?

Reply:
Oh, sorry, now I get it. You have used tabs for indentation in this line. Please use spaces instead 🙂